config.yaml

```diff
@@ -57,3 +57,13 @@ options:
       URL where one or more wasm runtime files can be found.
       The file can either be a .wasm file or a tar.gz archive containing one or more wasm runtime files.
       If set, the downloaded files(s) will be in /home/polkadot/wasm and the service argument `--wasm-runtime-overrides=/home/polkadot/wasm` will be used.
+  mnemonic-secret-id:
+    type: secret
+    description: |
+      The wallet to use for setting a session key on chain. This is the wallet that will sign the transaction when moving a validator/collator to this node
+      using the action 'start-validating'. The mnemonic needs to be added to the model as a secret with the key name 'mnemonic'. If this is a proxy account, the 'proxy-type' must be provided as well.
+      An application needs to be granted access to the secret to be able to use it.
+      Example:
+      juju add-secret validator1 mnemonic="foo bar" proxy-type="Staking" --info 14MofzwMbLm1JeBxLi2BKBRHXJce87DE5UWnttgGBBrQcEy9
```

Comment:
I made the proxy type configurable instead since that seems to differ a lot between chains.
`proxy-type="Staking"`